jupyter_config.json

```diff
@@ -1,13 +1,20 @@
 {
+    "Voila": {
+        "tornado_settings": {
+            "headers": {
```

Comment:
does this work or not?

i.e. can these options be removed from the `run.sh` script?

Reply:
This does indeed seem to work.

Reply:
The issue here being that this JSON file is actually never used when running on dokku, unless it is first copied to the local Jupyter config dir and renamed to `voila.json`.

Reply:
I will try and instead use the following option:

```console
--Voila.config_file_paths=<list-item-1>...
    Paths to search for voila.(py|json)
    Default: []
```

Reply:
Great!